First/Storage/AibidiaDatabaseContext.cs

```diff
@@ -17,7 +18,8 @@ public class AibidiaDatabaseContext : DbContext
 
         public AibidiaDatabaseContext()
         {
-            DbPath = @$"{AppDomain.CurrentDomain.BaseDirectory}Database\Aibidia_Tables.db";
+            string[] paths = {AppDomain.CurrentDomain.BaseDirectory, @"Database", "Aibidia_Tables.db"};
```

Comment:
Why is there  an @-symbol?? *angry*

Reply:
Ouch! My bad! So sorry! Fixed now! Please check again!